migrator/postgreshelper/load.go

```diff
@@ -52,14 +52,23 @@ func Load(conf *config.Config) (*gorm.DB, error) {
 				return
 			}
 		}
+		adminConfig.ConnConfig.Database = activeDB
 
 		// Add the active database and password to the source
-		source := fmt.Sprintf("%s password=%s database=%s client_encoding=UTF-8", conf.CentralDB.Source, password, activeDB)
-		source = pgutils.PgxpoolDsnToPgxDsn(source)
+		gormSource := fmt.Sprintf("%s password=%s database=%s client_encoding=UTF-8", conf.CentralDB.Source, adminConfig.ConnConfig.Password, activeDB)
```

Comment:
Does it make sense to put the db connection building the source, etc in a package that is just always shared between the migrator and central. We may want to tweak some parameters, but I think it would be nice to only have to change that in one place

Reply:
The migrator would manage the database selection. And central may not need Gorm in the future. The need would diverge anyway.